### ISSUE TITLE
Serializer not emitting JSON API spec

### ISSUE THREAD

**Issue by kjs3** [rails-api/active_model_serializers#1847]
#### Expected:

In Rails, when configured with `ActiveModelSerializers.config.adapter = :json_api`

I expected using `ArticleSimpleSerializer.new(article).as_json` (or `.to_json`) **_outside_** of a controller to emit JSON in JSON API format including `data`, `type`, and `attributes` root keys.

#### Actual:
I get JSON object with Serializer attributes. Essentially what I would have expected in 
`json["data"]["attributes"]`

#### Steps to reproduce

I have a serializer for an Article model but named `ArticleSimpleSerializer` so automatic lookup with…

```ruby
ActiveModelSerializers::SerializableResource.new(article)
```

… doesn't work.

Rendering in a controller _does_ emit JSON API format.

```ruby
render status: :ok, json: @article, serializer: ArticleSimpleSerializer
```

I can't seem to get this same output outside of a controller.

#### Environment

ActiveModelSerializers Version 0.10.2: 

ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin15]: 

Mac OS 10.11.5

Rails 4.2

**Comment by NullVoxPopuli** [comment]
`SerializableResource` will take all the render options, so you can pass the serializer to it :-)

Lemme know if that changes anything :-)

**Comment by kjs3** [comment]
Thanks @NullVoxPopuli!

```ruby
ActiveModelSerializers::SerializableResource.new(article, serializer: ArticleSimpleSerializer).as_json
```

works perfectly.

Are you interested in a PR that spells this out in the docs?

**Comment by NullVoxPopuli** [comment]
yeah, it looks like https://github.com/rails-api/active_model_serializers/blob/master/docs/howto/outside_controller_use.md doesn't say what can be in `options` :-)
